Benchmarks/Benchmarks/NIOPosixBenchmarks/Benchmarks.swift

```diff
@@ -64,4 +65,45 @@ let benchmarks = {
         )
     }
     #endif
+
+    Benchmark(
+        "MTELG.scheduleTask(in:_:)",
+        configuration: Benchmark.Configuration(
+            metrics: [.mallocCountTotal, .cpuTotal],
+            scalingFactor: .kilo
+        )
+    ) { benchmark in
+        let group = MultiThreadedEventLoopGroup(numberOfThreads: 1)
+        defer { try! group.syncShutdownGracefully() }
+        let loop = group.next()
+
+        benchmark.startMeasurement()
+        for _ in benchmark.scaledIterations {
+            loop.scheduleTask(in: .hours(1), {})
+        }
+        benchmark.stopMeasurement()
```

Comment:
FWIW: `stopMeasurement()` is redundant at the end of a benchmark (it's automatically called)

Reply:
I think this was a hangover from when I was playing around with a lower sample size and FWIW, it did make a minor difference having it called inside the closure that benchmark takes.

In any event, I've now removed them in bbf8f9f81.